python/src/rest_api/rest_api_impl.py

```diff
@@ -112,16 +111,24 @@ def _write_openapi_spec_to_file(output_directory: str) -> str:
     """
     full_file_path = os.path.join(output_directory, "AaC_OpenAPI_Schema.json")
 
-    with open(full_file_path, "w") as output_file:
-        json.dump(
-            get_openapi(
-                title=app.title,
-                version=app.version,
-                openapi_version=app.openapi_version,
-                description=app.description,
-                routes=app.routes,
-            ),
-            output_file,
-        )
-
-    return f"Successfully wrote the OpenAPI spec to {full_file_path}."
+    try:
+        with open(full_file_path, "w") as output_file:
+            json.dump(
+                get_openapi(
+                    title=app.title,
+                    version=app.version,
+                    openapi_version=app.openapi_version,
+                    description=app.description,
+                    routes=app.routes,
+                ),
+                output_file,
+                indent=4,
+            )
+
+        msg = "Successfully wrote the OpenAPI spec to {full_file_path}."
```

Comment:
```suggestion
        msg = f"Successfully wrote the OpenAPI spec to {full_file_path}."
```
This got dropped in the shift to returning two parts.